terra_notebook_utils/cli/workflows.py

```diff
@@ -122,7 +127,7 @@ def print_divider(self):
         print("-" * self.width)
 
     def ff(self, val: Any, width: int, decimals: int=2) -> str:
```

Comment:
Should `val: Any` be `Union[str, bool, int, float]`?

Reply:
Probably, but that should be handled outside of this PR :)